cdap-distributions/bin/build_yum_repo.sh

```diff
@@ -47,7 +47,7 @@ STAGE_DIR=${TARGET_DIR}/yumrepo
 
 S3_BUCKET=${S3_BUCKET:-repository.cask.co}
 S3_REPO_PATH=${S3_REPO_PATH:-centos/6/x86_64/cdap} # No leading or trailing slashes
-VERSION=${VERSION:-$(basename ${TARGET_DIR}/cdap-*.rpm | cut -d- -f2)}
+VERSION=${VERSION:-$(ls -1 *.rpm | head -n 1 | rpm --queryformat "%{VERSION}" -qp)} # query package file
```

Comment:
you sure this works?  I didn't think it could read from stdin:

```
$ rpm -qa | grep cloudera | head -n 1 | rpm --queryformat "%{VERSION}" -qp
rpm: no arguments given for query
```